### DISCUSSION TITLE
Onboarding PR not created

### DISCUSSION THREAD

**Discussion by bcaudan** [renovatebot/renovate#19578] [Request Help] [answered]
Hi,

I'd like to onboard the `datadog/browser-sdk` repository, when I update Renovate's repository access to add this repository it says the renovate was updated for the account. However, no onboarding PR is created and I can't see the repository from the dashboard.

By creating a new repository and granting renovate access from there, it seems to work fine.
By creating a new repository and afterwards granting the access from the Renovate app configuration, it does not seem to be taken into account.

Can you help me with that?

**Comment by rarkins** [comment, marked as answer]
Do you install Renovate using "All repositories", or "Select repositories"? If the latter, can you remove this repo and save, wait 10 seconds, then edit the list to add it back again? This should trigger a "new install" webhook

**Reply by bcaudan** [reply]
I use "Select repositories" but it is not added to the list after I select it and update repository access.

**Reply by rarkins** [reply]
What you're describing then is unfortunately a GitHub bug - suggest you screen record then raise a support request to them

**Reply by bcaudan** [reply]
OK, will do! Thanks for your answer.

**Reply by LiuVII** [reply]
@bcaudan did you happen to figure out the issue with Github?

**Reply by bcaudan** [reply]
I opened an issue on Github side and the next day the onboarding PR was created, no clue what happened.
